src/ui/services/staticService.ts

```diff
@@ -4,1167 +4,2207 @@
+            { id: 1058, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1058.png" },
+            { id: 1059, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1059.png" },
+            { id: 1060, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1060.png" },
+            { id: 1061, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1061.png" },
+            { id: 1062, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1062.png" },
+            { id: 1063, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1063.png" },
+            { id: 1064, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1064.png" },
+            { id: 1065, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1065.png" },
+            { id: 1066, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1066.png" },
+            { id: 1067, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1067.png" },
+            { id: 1068, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1068.png" },
+            { id: 1069, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1069.png" },
+            { id: 1070, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1070.png" },
+            { id: 1071, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1071.png" },
+            { id: 1072, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1072.png" },
+            { id: 1073, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1073.png" },
+            { id: 1074, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1074.png" },
+            { id: 1075, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1075.png" },
+            { id: 1076, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1076.png" },
+            { id: 1077, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1077.png" },
+            { id: 1078, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1078.png" },
+            { id: 1079, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1079.png" },
+            { id: 1080, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1080.png" },
+            { id: 1081, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1081.png" },
+            { id: 1082, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1082.png" },
+            { id: 1083, iconURL: "http://ddragon.leagueoflegends.com/cdn/7.2.1/img/profileicon/1083.png" },
+            { id: 1084, iconURL: "http://ddragon.leagueofleg
```

Comment:
This entire list is redundant, since every URL follows the same format (and as such, the urls can be computed when needed). Also, the version is hardcoded to `7.2.1`, either make this dynamic or use `4.20.1`.